.env.development

```diff
@@ -1,4 +1,4 @@
-REACT_APP_NHOST_BACKEND_URL=https://rgkjmwftqtbpayoyolwh.nhost.run/
```

Comment:
Removed? This env is used other places also.

Reply:
I haven't removed the complete variable
I have removed the extra `/` at the last.

Reply:
Yeah this is right. Resolving the comment.